package.json

```diff
@@ -49,7 +49,7 @@
     "react-native-static-server": "^0.4.2",
     "react-native-version-number": "^0.3.6",
     "react-native-video": "file:react-native-video-5.1.0-alpha2.tgz",
-    "react-native-webview": "^7.4.3",
+    "react-native-webview": "^11.0.2",
```

Comment:
damn 3 majors??!? wewwwwww. hopefully this fixes a bunch of stuff. excited to get this on testflight.

Reply:
🤞 this fixes you know what

Reply:
who shall not be named